plugins/Over9000/__init__.py

```diff
@@ -0,0 +1,42 @@
+﻿from random import choice, randint
+import discord
+from discord.ext import commands
+
+from .quotes import DBZQuotes
+
+from Plugin import AutomataPlugin
+
+
+class Over9000(AutomataPlugin):
+    """Kakarot"""
+
+    @commands.command()
+    async def over9000(self, ctx: commands.Context, dbz_char: str = 'goku'):
+        """Replies with a super saiyan emoji and random
+        Goku quote!
+
+        Takes character name as an argument.
+
+        """
+
```

Comment:
Too many blank lines. This codebase is formatted with black, so using black should re-format this file for you.

Black is usable as a formatter for the Python VSCode extension, if you happen to be a VSCode user ([VSCode docs here](https://code.visualstudio.com/docs/python/editing#_formatting)), but is pretty simple to use from the command line otherwise ([instructions for that here](https://black.readthedocs.io/en/stable/installation_and_usage.html))

Reply:
Fixed